simplevalue.go

```diff
@@ -0,0 +1,10 @@
+package cbor
+
+import "reflect"
+
+// SimpleValue represents CBOR simple value [0, 255].
```

Comment:
Add more info because CBOR simple value is not as simple as it's name.

- CBOR simple value is an extension point like CBOR tag.

- CBOR simple value is a subset of major type 7 that isn't floating-point. 

- CBOR simple value is "identified by a number between 0 and 255, but distinct from that number itself".

- CBOR simple values identified by 20..23 are:  "false", "true" , "null", and "undefined".   Other CBOR simple values are currently unassigned/reserved by IANA.

Reply:
Good suggestion.  Added comment in commit 2e7844124ae09cb451d66b8e95e5c04a2be577ae.